src/cmake-build-debug/include/Globals.h

```diff
@@ -0,0 +1,2 @@
+/* This file allows Globals.h to be included with an absolute path */
+#include "/Users/Lode/dev/Github/cuberite/src/Globals.h"
```

Comment:
Get this file out of the PR. I would suggest rebasing (or just commit the removal and squash all commits, when the PR works).

To prevent this problem in the future, move the `cmake-build-debug` folder outside of the Git repository or into a directory named `build` (as the directory build is already in the gitignore file, so any subdirectory is ignored :smirk: )